DOC : update documentation with the new carpetplot (#1045)
nipreps/mriqc: docs/source/reports/bold.rst

   signal intensity at the extremes for the identification
   of spikes, the outliers metric, the DVARS and the
   :abbr:`FD (framewise displacement)`. Finally the
   so-called carpetplot [Power2016]_. 

>    The carpet plot rows, corresponding to voxel timeseries,

Reviewer: ```suggestion
   The carpet plot rows correspond to voxelwise time series,
```

>    are separated into regions : cortical gray-matter, deep 

Reviewer: ```suggestion
   and are separated into regions: cortical gray-matter, deep 
```

>    gray-matter, white-matter and cerebrospinal fluid, cerebellum 

Reviewer: ```suggestion
   gray-matter, white matter and cerebrospinal fluid, cerebellum 
```

>    and crown [Provins2022]_. The crown corresponds to the voxels located on a 

Reviewer: ```suggestion
   and the brain-edge or “crown” [Provins2022]_.
   The crown corresponds to the voxels located on a 
```

   signal intensity at the extremes for the identification
   of spikes, the outliers metric, the DVARS and the
   :abbr:`FD (framewise displacement)`. Finally the
   so-called carpetplot [Power2016]_. 
   The carpet plot rows correspond to voxelwise time series,

>    and are separated into regions: cortical gray-matter, deep 

Author: ```suggestion
   and are separated into regions: cortical gray matter, deep 
```

>    gray-matter, white matter and cerebrospinal fluid, cerebellum 

Author: ```suggestion
   gray matter, white matter and cerebrospinal fluid, cerebellum 
```

.. topic:: References

  .. [Patriat2015] Patriat, R., EK Molloy, RM Birn, T. Guitchev, and A. Popov. “Using Edge Voxel Information to 
     Improve Motion Regression for Rs-FMRI Connectivity Studies.” Brain Connectivity 5, no. 9 (28 2015): 582–95. 

>      doi: `10.1089/brain.2014.0321 <https://doi.org/10.1089/brain.2014.0321>`_.

Reviewer: ```suggestion
     doi: `10.1089/brain.2014.0321 <https://doi.org/10.1089/brain.2014.0321>`__.
```

Reviewer: Two underscores is the right choice here.

Author: @oesteban what do the one underscore or two underscores do ?

Reviewer: LMGTFY...

https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use

  .. [Power2016] Power JD, A simple but useful way to assess fMRI scan qualities.

>      NeuroImage. 2016. doi: `10.1016/j.neuroimage.2016.08.009 <http://doi.org/10.1016/j.neuroimage.2016.08.009>`_

Reviewer: ```suggestion
     NeuroImage. 2016. doi: `10.1016/j.neuroimage.2016.08.009 <http://doi.org/10.1016/j.neuroimage.2016.08.009>`__.
```

  .. [Provins2022] Provins, Céline, Christopher J. Markiewicz, Rastko Ciric, Mathias Goncalves, César Caballero-Gaudes, 
     Russell Poldrack, Patric Hagmann, and Oscar Esteban. “Quality Control and Nuisance Regression of FMRI, Looking out 

>      Where Signal Should Not Be Found.” OSF Preprints, January 21, 2022. doi: `10.31219/osf.io/hz52v <https://doi.org/10.31219/osf.io/hz52v>`_

Reviewer: ```suggestion
     Where Signal Should Not Be Found.” OSF Preprints, January 21, 2022.
     doi: `10.31219/osf.io/hz52v <https://doi.org/10.31219/osf.io/hz52v>`__.
```
